docs/pages/enterprise/workflow/ssh-approval-slack.mdx

```diff
@@ -228,23 +248,22 @@ $ tsh login --request-roles=dba
 
 #### Approve or deny the request on Slack
 
-The messages should automatically get updated to reflect the action you just clicked. You can also check the request status with `tctl`:
+The messages should automatically get updated to reflect the action you just clicked. You can also check the request status with `tsh`:
 
 ```bash
-$ tctl request ls
+$ tsh request ls
 ```
 
 ### TSH User Login and Request Admin Role
 
 You can also test the full workflow from the user's perspective using `tsh`:
 
 ```bash
-# tsh login --request-roles=REQUESTED_ROLE
+# tsh request new --request-roles=REQUESTED_ROLE
 Seeking request approval... (id: 8f77d2d1-2bbf-4031-a300-58926237a807)
 ```
 
-You should now see a new request in Teleport, and a message about the request on Slack. You can
-approve or deny it and `tsh` should login successfully or error out right after you click an action button on Slack.
+You should now see a new request in Teleport, and a message about the request on Slack. Based on your open source or licensed version you will get CLI or web console instructions on approving/denying the request.
```

Comment:
What's the difference between the two?

Reply:
licensed allows approving/denying in the web console.  reworded.